src/Features.php

```diff
@@ -304,11 +189,11 @@ public static function on_field_save( $field_id, $updated, $action, $field ) {
 		// This requires a toggle because we may be hitting a sort of rate limit from the deploy purge alone.
 		// For now it's better to leave this off on test instances, to avoid purges failing on production because we hit
 		// the rate limit.
-		if ( ! self::is_active( self::PURGE_ON_FEATURE_CHANGES ) ) {
+		if ( ! PurgeOnFeatureChanges::is_active() ) {
 			return;
 		}
 
-		if ( in_array( $field_id, [ self::NEW_DESIGN_COUNTRY_SELECTOR, self::NEW_DESIGN_NAVIGATION_BAR ], true ) ) {
+		if ( in_array( $field_id, [ NewDesignCountrySelector::id(), NewDesignNavigationBar::id() ], true ) ) {
```

Comment:
In a follow up we can move this partly into the classes, using a method `needs_cache_purge`. We could even go further and prevent full cache purges, in case a feature, for example only affects 1 post type.

But keeping this out of the PR for now to reduce the scope of changes.